java/com/google/re2j/JavaCategoryTables.java

```diff
@@ -0,0 +1,815 @@
+
+package com.google.re2j;
+
+// AUTOGENERATED by MakeJavaCategories.java - do not modify
```

Comment:
Is this the case? The existing [unicode tables](https://github.com/google/re2j/blob/master/java/com/google/re2j/UnicodeTables.java) are autogenerated and are checked in. Generating this file during the build would probably involve writing a Maven plugin.